data/api/client-server/rooms.yaml

```diff
@@ -105,13 +105,30 @@ paths:
           example: ""
           schema:
             type: string
+        - in: query
```

Comment:
this should have the `x-addedInMatrixVersion` or whatever that is called, if we're classifying it as a `.feature`